### DISCUSSION TITLE
Option to enable "result" messages using device groups?

### DISCUSSION THREAD

**Discussion by dkotte** [arendst/Tasmota#12833] [Ideas]
Hi all,
is there an option to get the result messages (via MQTT) while working with device groups?
Currently all devices have the same color or brightness, but the control-system shows the "old" data for the secondary devices, because i got not result (or other) messages from them.

**Comment by happy2rv** [comment]
I'm observing the same thing.   Using a Shelly RGBW2, when Dimmer is changed by devgroup, no messages are sent to mqtt.  If I change the value using command, mqtt, or the web interface, there is always a result mqtt message.  I can solve this by creating a rule "on Dimmer#state do Dimmer"  which triggers a Result message, but it seems like this should be default behavior. 

Are we missing something, or is there an option for this?

**Reply by sfromis** [reply]
While you can get a more complete status reporting having the rule issue the command `HsbColor` instead of `Dimmer`, this still does not provide reporting of all updates, like changing color saturation, and there is no option for this. Still, power on/off does result in messages when shared.

I suppose that a point of "less detailed" reporting could be that with lights bound in a group, it is not very useful to have a control system keep track of state of each, and it would generate quite a bit of traffic with big groups.

**Reply by happy2rv** [reply]
I can appreciate "squelching" the noise, but if there is no update for a device, I'm not sure how you would keep a control system like OpenHAB or HomeAssistant in sync without some kind of additional rules and tracking each item for changes to trigger those rules.  For instance, lets take the simple case I have setup for testing.  I have a Shelly RGBW2 that I'm only using for 1 channel of white LED strip lights, a Martin Jerry Tasmota dimmer. 

You are correct, I don't care about each individual item in a group's state.  I care about the group's state which should match accross the group.  So in my test case, only the RGBW2 is defined in OpenHAB.  However, if I change the brightness from the MJ dimmer, updating the group, the state change doesn't register in OpenHAB.  In order to track the change from OpenHAB without the additional rule, I would have to monitor each device in the group and then sync everything to the device that changed.  This kind of defeats any value of groups with respect to the controlling application.  There's still value in that the groups work even if the controlling app is offline, but it's far from ideal.
